devops/transform_circle.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+
+import yaml
+
+
+def convert(compose_config, container_name):
+    for service in compose_config.get('services', {}).values():
+        if 'volumes' in service:
+            service['volumes_from'] = [f'container:{container_name}']
+            del service['volumes']
+
```

Comment:
Nice, but at the same time it's annoying that there's no clean way to do this within the Docker config files themselves.

Reply:
Yeah, we're relying on synchronous file mounts when running locally, which isn't feasible with the networked Docker. I wonder if there's a clean way to avoid that? Maybe some bash script that copies the local files before running? Dev cycles would be slower (we'd have to restart and reset files after every change), but file access would be quicker...